mbuild/conversion.py

```diff
@@ -716,6 +778,65 @@ def from_pybel(pybel_mol,
     return compound
 
 
+def from_rdkit(rdkit_mol,
+               compound=None,
+               coords_only=False,
+               smiles_seed=0):
+    """Return an mbuild Compound based on a smiles string using RDKit.
+
+    Parameters
+    ---------
+    rdkit_mol : rdkit.Chem.rdchem.Mol
+        RDKit mol to generate an mBuild compound
+    compound : mb.Compound, optional, default=None
+        The host mbuild Compound.
+    coords_only : bool, optional, default=False
+        Set preexisting atoms in compound to coordinates given
+        by structure. Note: Not yet implemented, included only
+        for parity with other conversion functions.
+    smiles_seed : int, optional, default=0
+        Random number seed for PRNG, set to -1 for non-deterministic behavior
+
+    Returns
+    -------
+    mbuild.Compound
+
+    NOTES
+    -----
+    Option `coords_only` currently is not implemented, it is only provided to
+        maintain parity with other conversion methods.
+
+    """
+    from rdkit import Chem
+    from rdkit.Chem import AllChem
+
+    mymol = Chem.AddHs(rdkit_mol)
+    AllChem.EmbedMolecule(mymol, randomSeed=smiles_seed)
```

Comment:
```suggestion
    AllChem.EmbedMolecule(mymol, randomSeed=smiles_seed)
    AllChem.UFFOptimizeMolecule(mymol)
```
@chrisjonesBSU  mentioned that some non ideal conformations can be generated via smiles depending on the seed. I found adding this step maintains the same particle positions with optimized relative orientations.